src/tests/state/status/selectors.test.js

```diff
@@ -0,0 +1,47 @@
+import { fromJS } from "immutable";
+import { getStatus } from "../../../state/status/selectors";
+
+describe("Status selectors", () => {
+    const getMockedState = status => {
+        return fromJS({
+            status: {
+              selectedStatus: status
+            }
+        });
+    }
+
+    it("should get draft status", () => {
```

Comment:
Ah okay so, this is a case where, instead of thinking about this from a perspective of 'what are all the states my data can be in', try to think about it from the perspective of 'what are all the states my code cares about'.  Does the code have explicitly separate cases for handling each of these statuses?  How many cases does the code think about it? And then how many tests do you need to test those cases?

Reply:
oooooo okies makes sense, I went a bit overboard with my tests lolol